### DISCUSSION TITLE
Support for NavIC

### DISCUSSION THREAD

**Discussion by notparthreally** [Akylas/alpimaps#18] [Ideas]
It would be really great if you could add support for the Indian sat cluster. It is named NavIC.

**Comment by farfromrefug** [comment]
@notparthreally the support is already partially here. NavIC satelilites will be shown in the satellite view. However they might not be listed in the bottom horizontal grid. The reason is that i miss info about them. And i cant seem to find it on the web.
I need to get their PRN/NMEA ID. Those are unique numbers defining them.
It is either that or i need to find a new way of representing the satellite type (replacing the horizontal bottom grid view).

**Reply by notparthreally** [reply]
Thanks for responding mate. I will look at it and find anything that I can help you with.
